CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 
 
 * chore: bump `cairo-lang-` dependencies to 2.7.1 [#1823](https://github.com/lambdaclass/cairo-vm/pull/1823)
+* feat: Implement `SECP related` hints [#1829](https://github.com/lambdaclass/cairo-vm/pull/1829)
```

Comment:
This change should be in the "Upcoming Changes" section

Reply:
Resolved by https://github.com/lambdaclass/cairo-vm/pull/1829/commits/6e5227c51cb8b2f0b136abe8327fe4373390f2b2